### PR TITLE
Sanity Test Support for Unity Manila Shares

### PR DESCRIPTION
**Pull request by rajinir**

### CODE REVIEW

**Comment by rajinir** [comment]
Tested them on my stamp. These are executed only if manila  is deployed

File: src/pilot/deployment-validation/sanity_test.sh [hunk 1]
```diff
@@ -642,6 +643,80 @@ setup_cinder(){
   done
 }
 
+setup_manila(){
+  info "### Manila test"""
+  
+  set_admin_scope
+  manila_exists=$(manila service-list | grep manila-share |  head -n 1  | awk '{print $4}')
+  echo "$manila_exists";
```
Comment (jtwill98) [inline src/pilot/deployment-validation/sanity_test.sh]: No need for this echo line

File: src/pilot/deployment-validation/sanity_test.sh [hunk 2]
```diff
@@ -642,6 +643,80 @@ setup_cinder(){
...
+  execute_command "manila share-network-list"
+  
+	  
+  execute_command "manila list"
+
+  info "### Kicking off share creation..."
+
+  shares=()
+  share_name=$BASE_SHARE_NAME
+  echo "Share Name = $share_name";
```
Comment (jtwill98) [inline src/pilot/deployment-validation/sanity_test.sh]: no need for this line

File: src/pilot/deployment-validation/sanity_test.sh (hunk 1 above)
Comment (rajinir) [reply src/pilot/deployment-validation/sanity_test.sh]: Will remove. I guess I had these for debugging

File: src/pilot/deployment-validation/sanity_test.sh (hunk 2 above)
Comment (rajinir) [reply src/pilot/deployment-validation/sanity_test.sh]: Will remove. I guess I had these for debugging

**Pull request merged**